sigmastate/src/main/scala/sigmastate/serialization/ValueSerializer.scala

```diff
@@ -347,6 +347,7 @@ object ValueSerializer extends SigmaSerializerCompanion[Value[SType]] {
     }
   }
 
+  // TODO mainnet v5.0: control maxTreeDepth same as in deserialize
```

Comment:
Do GH issue, not todo

Reply:
added issue https://github.com/ScorexFoundation/sigmastate-interpreter/issues/781